app/components/TeamMemberList.tsx

```diff
@@ -0,0 +1,24 @@
+import { getTeamMembers } from '../../utils/teamMembers';
+import TeamCardMember from './TeamCardMember';
+
+export default function TeamMemberList() {
+  const teamMembers = getTeamMembers();
```

Comment:
You can import data file here, just fix the file path if needed:

```jsx
import teamMembers from '../data/team-information.json';
```

Then, after you import data file, you could remove the file `utils/teamMembers.ts`.